macros/support/src/interface/iid.rs

```diff
@@ -46,6 +46,7 @@ impl IID {
         let data4_8 = hex_lit(data4_8);
         quote!(
             #[allow(missing_docs)]
+            #[doc(hidden)]
```

Comment:
This seems less like an implementation detail than the other items. I can certainly imagine someone wanting to directly use an interfaces IID. Are we sure we want this to be hidden?